axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -993,6 +1008,184 @@
           "address": "axelar1g03zp32gqx6qg3z59csfj4rm56lkpyqkdw9lg4qdgg6uxyupryrqcycxt9"
         }
       },
+      "XrplMultisigProver": {
+        "xrpl-2025-q1": {
```

Comment:
shall we call this xrpl-test-1 inline with xrpl-evm-test-1 for the evm side chain ?